modules/gke-cluster-standard/main.tf

```diff
@@ -338,6 +338,9 @@ resource "google_container_cluster" "cluster" {
         exclusion_name = exclusion.value.name
         start_time     = exclusion.value.start_time
         end_time       = exclusion.value.end_time
+        exclusion_options {
```

Comment:
can you make this block dynamic based on wether `exclusion.value.scope` is null?

Reply:
Hi @juliocc yes will work on this and push the change.